src/components/details-view/details-view.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import capitalize from 'lodash/capitalize';
+
+import PaneHeader from '@folio/stripes-components/lib/PaneHeader';
+import Icon from '@folio/stripes-components/lib/Icon';
+
+import KeyValueLabel from '../key-value-label';
+import styles from './details-view.css';
+
+const cx = classNames.bind(styles);
+
+export default class DetailsView extends Component {
```

Comment:
it might be worth throwing a comment in about what this component does and generally how it works.  the line-by-line comments are helpful, but to someone coming in from the outside it may be difficult to parse what the intent is, how the scroll math/logic works and what things like `sticky` mean in the context of the component.  i had the benefit of riding along while you were writing some of this so i somewhat get the picture, but to someone with fresh eyes this is pretty intense.  it is also pretty clearly delineated in your PR description, so i guess it could be argued that people could look it up there.

Reply:
Good point. I thought I documented this, but guess not